### PR TITLE
Add SkyCube.sky_image()

### PR DESCRIPTION
**Pull request by OlgaVorokh**
As discussed in [PR582](https://github.com/gammapy/gammapy/pull/582)
There are no tests as the function tested in whole in test for PR528

### CODE REVIEW

File: gammapy/cube/core.py [hunk 1]
```diff
@@ -324,6 +324,22 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos, copy=True):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap` along slicepos.
+
+        Parameters
+        ----------
+        slicepos : int
+            Slice position
```
Comment (cdeil) [inline gammapy/cube/core.py]: I've started to use `idx` for "index" in other places.
So my suggestion here (and in other places for consistency) would be to use this name and description:
```
idx_energy : int
    Energy slice index
```

Also: please document the `copy` option in the docstring. (I know ... it's trivial ... but the simplest is just to always document all parameters without any thought / discussion when to document them or not.)

**Comment by cdeil** [comment]
This looks good to me.

But please do add a simple test for this.

At the moment it's rather trivial, but soon we want to add this: in `SkyMap.meta`, store the energy band information as keys `energy_min`, `energy_max` (in TeV) and `energy_index`.
Having this info available on the SkyMap object will help us with book-keeping in analysis methods that decompose cubes to images in energy bands (to be seen, discussed, but my guess is that this is a good place for this info).
So if there is a test for this method, it'll be easier to add this and test it.
@OlgaVorokh - If you want, you can do this addition to `map.meta` here and in the test assert that it's filled OK for one example. Or leave this for later, but please add the test already here.

**Comment by OlgaVorokh** [comment]
@cdeil I want you to see this!

File: gammapy/cube/core.py [hunk 2]
```diff
@@ -324,20 +324,22 @@ def solid_angle_image(self):
...
 
-    def sky_image(self, slicepos, copy=True):
+    def sky_image(self, idx_energy, copy=True):
         """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap` along slicepos.
 
         Parameters
         ----------
-        slicepos : int
-            Slice position
+        ind_energy : int
```
Comment (adonath) [inline gammapy/cube/core.py]: There's a minor typo: `ind_energy` --> `idx_energy`

Comment (cdeil) [inline gammapy/cube/core.py]: I fixed the typo in 6ac355f .

**Comment by cdeil** [comment]
I merged this into master just now.
Thanks!

**Comment by OlgaVorokh** [comment]
@cdeil Skycube doesn't has meta attribute, do you suggest constructing meta for resulting SkyCube from energy in the future?

Right now simple test "Take load_poisson_stats_image(), make skyimagelist, convert to skycube, take 0-th slice and compare with load_poisson_stats_image()" fails because neither skyimagelist nor skycube have meta parameter, can I remove meta equality assert for now?

**Comment by cdeil** [comment]
@OlgaVorokh - Good catch!

I think yes, [SkyCube](https://gammapy.readthedocs.io/en/latest/api/gammapy.cube.SkyCube.html) should probably have a `meta` dict, just like [SkyMap](http://docs.gammapy.org/en/latest/api/gammapy.image.SkyMap.html) does.

For SkyMap, `meta` contains the FITS header information on read / write (and in addition we can add / remove things in Gammapy for communicating information between analysis steps).

SkyCube objects are stored in two FITS HDUs though, there's a second BINTABLE one with the energy array (see https://gammapy.readthedocs.io/en/latest/_modules/gammapy/cube/core.html#SkyCube.read).
So it's less clear what to do for I/O.
My suggestion would be to read the data array FITS header and fill it in meta, just like it's done in SkyMap. The ENERGY HDU doesn't contain useful information in the header.

So @OlgaVorokh -- If you wan to take on this project of adding `meta` to SkyCube and changing it's FITS I/O to what I just described and maybe adapting calling sites and tests, that would be great!
Please do it in a separate PR.

**Pull request merged**